test_schemas/en/test_section_summary.json

```diff
@@ -187,6 +187,60 @@
                 }
             ]
         },
+        {
+            "id": "household-count-section",
+            "title": "Household member count",
+            "summary": {
+                "show_on_completion": true,
+                "title": {
+                    "text_plural": {
+                        "forms": {
+                            "one": "{number_of_people} person lives here",
+                            "other": "{number_of_people} people live here"
+                        },
+                        "count": {
+                            "source": "answers",
+                            "identifier": "number-of-people-answer"
+                        }
+                    },
+                    "placeholders": [
+                        {
+                            "placeholder": "number_of_people",
+                            "value": {
+                                "source": "answers",
+                                "identifier": "number-of-people-answer"
+                            }
+                        }
+                    ]
+                }
+            },
+            "groups": [
+                {
+                    "id": "number-of-people-group",
+                    "title": "Number of people",
+                    "blocks": [
+                        {
+                            "type": "Question",
+                            "id": "number-of-people",
+                            "question": {
+                                "answers": [
+                                    {
+                                        "id": "number-of-people-answer",
+                                        "label": "How many people live here?",
```

Comment:
This should be the question title, the answer label should be something like "Number of people".